tools/kdc/Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM mesosphere/kdc:latest@sha256:7d8fa40153fde988df54d2b1f3f25cab3e9eac4fb20ab454096765eab9161198
+# Build the kdc-api-server in a separate container
```

Comment:
Please use a different container for building and running the binary. We don't want to drag the toolchain around.
Also, please use a separate `Dockerfile` file for each.

Reply:
That's what I am doing 🙂 
Unless you are referring to the `mesosphere/kdc:latest` image...

Reply:
:man_facepalming: